CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## [5.0.5] - 2022-10-13
```

Comment:
if I'm not mistaken, this versions updated automatically.
In this case you do not need to write it.